requirements/requirements.txt

```diff
@@ -0,0 +1,10 @@
+pre-commit
+flake8
+black
+pytest
+pytest-cov
+mypy
+docformatter
+types-PyYAML
+types-requests
+dialoguekit
```

Comment:
Should we require a specific minimum version here?

Reply:
I was thinking about it as well. We could just to be safe and update the version if necessary. As long as the structure of DialogueKit stays the same it should work fine by just downloading the last version on pypi.

Reply:
OK, let's add a minumum version then and we update as necessary.

Reply:
Done.